devito/data/data.py

```diff
@@ -312,7 +312,7 @@ def index_dist_to_repl(idx, decomposition):
     Convert a distributed array index a replicated array index.
```

Comment:
Convert a distributed array index **to(?)** a replicated array index.

Reply:
fixing, thanks